digits/static/js/jquery.time_filters.js

```diff
@@ -22,12 +22,12 @@ function print_time_diff(diff) {
     }
 
     if (days >= 1)
-        return pair(days, 'day', hours, 'hour');
+        return pair(Math.floor(days), 'day', Math.round(hours), 'hour');
     else if (hours >= 1)
-        return pair(hours, 'hour', minutes, 'minute');
+        return pair(Math.floor(hours), 'hour', Math.round(minutes), 'minute');
     else if (minutes >= 1)
-        return pair(minutes, 'minute', seconds, 'second');
-    return plural(seconds, 'second');
+        return pair(Math.floor(minutes), 'minute', Math.round(seconds), 'second');
+    return plural(Math.round(seconds), 'second');
```

Comment:
I should probably stare at this for a little while longer but I'm missing the point behind the change :cold_sweat: